package.json

```diff
@@ -56,7 +56,7 @@
     "@storybook/react": "^4.0.0-alpha.8",
     "babel-cli": "^6.22.2",
     "babel-core": "^6.26.0",
-    "babel-eslint": "^8.2.2",
+    "babel-eslint": "^7.2.3",
```

Comment:
Why downgrade? Can you enhance the comments ideally in package.json but perhaps just in the PR explaining why we need an older babel-eslint?

Reply:
This is because of this issue: https://github.com/babel/babel-eslint/issues/530#issue-265804171

You can reproduce this by removing `node_modules` and run `npm` install again.

Reply:
I'm trying put something in place so our next `yarn upgrade --latest` doesn't cause us to miss this again.

Reply:
I think this was something with the latest babel-plugin release. downgrading to 7 should never install 8. So I feel we are relatively safe here.